source/process.adoc

```diff
@@ -296,11 +298,16 @@ To produce a Specification Version, a Specification Project must engage in a for
 include::diagrams/efsp_lifecycle.dot[]
 ----
 
-A Specification Project’s first Release Cycle starts with the successful completion of a Creation Review. To start a subsequent Release Cycle, the Specification Team presents a Release Plan to the Specification Committee in a Plan Review. The Plan Review must be approved by a Super-majority of the Specification Committee.
+A Specification Project’s first Release Cycle starts with the successful completion of a Creation Review. To start a subsequent Release Cycle (and every Release Cycle thereafter), the Specification Team must present a Release Plan to the Specification Committee in a <<efsp-reviews-plan,Plan Review>>. The Plan Review must be approved by a Super-majority of the Specification Committee. 
+
+A Release Cycle ends when the Specification Team delivers a Specification Version to the EMO and Specification Committee via a <<efsp-reviews-release,Release Review>>. To extend the Release Cycle, the Specification Team must stage a Milestone and engage in a <<efsp-reviews-progress, Progress Review>>. The Specification Team must engage in either a Release Review (which ends the Release Cycle) or a Progress Review (which extends the Release Cycle) within twelve (12) months of the last Review.
```

Comment:
It's the successful completion of the Release Review that ends the Release Cycle, right?
If the Release Review fails, the Release Cycle continues.

Reply:
Agreed.

In the event that a Release Review fails, the project team will need to mitigate highlighted issues and re-engage.